apps/antalmanac/src/components/RightPane/CoursePane/SearchForm/HelpBox.tsx

```diff
@@ -1,62 +1,62 @@
-    },
-    list: {
-        paddingLeft: '1.5em',
-    },
-    images: {
-        display: 'flex',
-        gap: 10,
-        overflow: 'auto',
-    },
-    image: {
-        height: 250,
-    },
 };
+
 interface HelpBoxProps {
     classes: ClassNameMap;
 }
 
 const HelpBox = ({ classes }: HelpBoxProps) => {
     return (
         <Paper variant="outlined" className={classes.container}>
-            <h2 className={classes.heading}>Need help planning your schedule?</h2>
-            <ol className={classes.list}>
-                <li>
-                    Browse undergraduate majors on the{' '}
-                    <a href="https://catalogue.uci.edu/undergraduatedegrees/" target="_blank" rel="noopener noreferrer">
-                        UCI Catalogue
-                    </a>
-                    .
-                </li>
-                <li>Select your major.</li>
-                <li>
-                    View the &quot;REQUIREMENTS&quot; and &quot;SAMPLE PROGRAM&quot; tabs to see what classes you should
-                    take.
-                </li>
-            </ol>
-            <div className={classes.images}>
-                <img
-                    className={classes.image}
-                    src="/helpbox1.png"
-                    alt='UCI General Catalogue with "Explore Undergraduate Programs" button highlighted'
-                />
-                <img className={classes.image} src="/helpbox2.png" alt="Undergraduate Majors and Minors page" />
-                <img
-                    className={classes.image}
-                    src="/helpbox3.png"
-                    alt='Electrical Engineering page with "REQUIREMENTS" and "SAMPLE PROGRAM" tabs highlighted'
-                />
-            </div>
+            <Typography variant="h5" fontWeight={'bold'}>
+                Need help planning your schedule?
+            </Typography>
+            <Typography>
+                <ol>
+                    <li>
+                        Browse undergraduate majors on the{' '}
+                        <a
+                            href="https://catalogue.uci.edu/undergraduatedegrees/"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            UCI Catalogue
+                        </a>
+                        .
+                    </li>
+                    <li>Select your major.</li>
+                    <li>
+                        View the &quot;REQUIREMENTS&quot; and &quot;SAMPLE PROGRAM&quot; tabs to see what classes you
+                        should take.
+                    </li>
+                </ol>
+            </Typography>
+            <ImageList gap={10} cols={3}>
+                <ImageListItem>
+                    <img
+                        src="/helpbox1.png"
+                        alt='UCI General Catalogue with "Explore Undergraduate Programs" button highlighted'
+                    />
```

Comment:
```suggestion
                    <Box
                       component="img"
                        src="/helpbox1.png"
                        alt='UCI General Catalogue with "Explore Undergraduate Programs" button highlighted'
                    />
```
ngl, this is kinda cursed, so maybe it's not worth doing lol.

Reply:
```typescript
        <ImageListItem key={item.img}>
          <img
            src={`${item.img}?w=164&h=164&fit=crop&auto=format`}
            srcSet={`${item.img}?w=164&h=164&fit=crop&auto=format&dpr=2 2x`}
            alt={item.title}
            loading="lazy"
          />
        </ImageListItem>
```

This is how MUI demos `ImageListItem`, so I think I'll pass on the cursed box 👻